### PR TITLE
AuthZ instead of AuthN

### PR DESCRIPTION
**Pull request by michaellyoungg**
Page improperly uses AuthN terminology instead of AuthZ

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
No, the article is correct. Look at the API
```
.RequireAuthenticatedUser()
```

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/security/authorization/secure-data.md [hunk 1]
```diff
@@ -102,19 +102,19 @@ Append <xref:Microsoft.AspNetCore.Identity.IdentityBuilder.AddRoles%2A> to add R
...
+* Is applied to all requests that do not explicitly specify an authorization policy. For requests served by endpoint routing, this would include any endpoint that does not specify an authorization attribute. For requests served by other middleware after the authorization middleware, such as [static files](xref:fundamentals/static-files), this would apply the policy to all requests.
 
-Setting the fallback authentication policy to require users to be authenticated protects newly added Razor Pages and controllers. Having authentication required by default is more secure than relying on new controllers and Razor Pages to include the `[Authorize]` attribute.
+Setting the fallback authorization policy to require users to be authenticated protects newly added Razor Pages and controllers. Having authorization required by default is more secure than relying on new controllers and Razor Pages to include the `[Authorize]` attribute.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:117]:
```suggestion
Setting the fallback authorization policy to require users to be authenticated protects newly added Razor Pages and controllers. Having authentication required by default is more secure than relying on new controllers and Razor Pages to include the `[Authorize]` attribute.
```

File: aspnetcore/security/authorization/secure-data.md [hunk 2]
```diff
@@ -102,19 +102,19 @@ Append <xref:Microsoft.AspNetCore.Identity.IdentityBuilder.AddRoles%2A> to add R
...
 
 <xref:Microsoft.AspNetCore.Authorization.AuthorizationPolicyBuilder.RequireAuthenticatedUser%2A> adds <xref:Microsoft.AspNetCore.Authorization.Infrastructure.DenyAnonymousAuthorizationRequirement> to the current instance, which enforces that the current user is authenticated.
 
-The fallback authentication policy:
+The fallback authorization policy:
 
-* Is applied to all requests that do not explicitly specify an authentication policy. For requests served by endpoint routing, this would include any endpoint that does not specify an authorization attribute. For requests served by other middleware after the authorization middleware, such as [static files](xref:fundamentals/static-files), this would apply the policy to all requests.
+* Is applied to all requests that do not explicitly specify an authorization policy. For requests served by endpoint routing, this would include any endpoint that does not specify an authorization attribute. For requests served by other middleware after the authorization middleware, such as [static files](xref:fundamentals/static-files), this would apply the policy to all requests.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/secure-data.md:115]:
```suggestion
* Is applied to all requests that don't explicitly specify an authorization policy. For requests served by endpoint routing, this includes any endpoint that doesn't specify an authorization attribute. For requests served by other middleware after the authorization middleware, such as [static files](xref:fundamentals/static-files), this applies the policy to all requests.
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by serpent5 — APPROVED** [review]

**Comment by serpent5** [comment]
Thanks for your contribution, @michaellyoungg. Be sure to check [What's New](https://docs.microsoft.com/en-us/aspnet/core/whats-new) early next month, where you'll see yourself listed as a valued contributor.

**Pull request merged**